src/Service.Tests/DatabaseSchema-MsSql.sql

```diff
@@ -252,7 +253,17 @@ CREATE TABLE mappedbookmarks
 (
 	id int IDENTITY(1,1) PRIMARY KEY,
 	bkname nvarchar(50) NOT NULL
-) 
+)
+
+create table books_sold
+(
+  [id] int PRIMARY KEY not null,
+  [book_name] varchar(50),
+  [row_version] timestamp,
```

Comment:
Do we even support timestamp datatype? Given than `time`datatype is not checked in yet. What does this `timestamp` type get mapped to?

Reply:
`time` relates to time. `timestamp` is a column related to versioning.
https://learn.microsoft.com/en-us/sql/relational-databases/xml/for-xml-support-for-the-timestamp-data-type?view=sql-server-ver16

Reply:
From the above article:

`In the FOR XML transformation, timestamp type values are treated as varbinary(8) data and will always be base 64 encoded. The XSD or XDR schema, if requested, reflects this type.`

Reply:
nit: the linked article is specific to using `FOR XML` transformation. For explicit details about how `timestamp` is treated, you can reference the `rowversion` documentation. https://learn.microsoft.com/en-us/sql/t-sql/data-types/rowversion-transact-sql?view=sql-server-ver16